skimage/morphology/_flood_fill.py

```diff
@@ -45,10 +45,15 @@ def flood_fill(image, seed_point, new_value, *, selem=None, connectivity=None,
         value of `image` at `seed_point` to be filled.  This is fastest.
         If a tolerance is provided, adjacent points with values within plus or
         minus tolerance from the seed point are filled (inclusive).
-    inplace : bool, optional
+    in_place : bool, optional
         If True, flood filling is applied to `image` inplace.  If False, the
```

Comment:
Our mistake from before, but note that there is no such English word as "inplace": it should be "in-place".

Reply:
thank you for the review :+1: this is fixed now.